.gitignore

```diff
@@ -134,4 +134,4 @@ dmypy.json
 data/
 docker-compose.override.yml
 .vscode/
-src/media/
+src/media/chains
```

Comment:
What's the reasoning behind this change? I think we need to gitignore all the local files in `src/media/safe-apps`. If the intention is to add the default logo (`icon_url.jpg`) to the repository, we could use the `!` prefix to whitelist 🤔